src/components/Templates/GetNfscCard/GetNfscCard.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { Chip } from '../../Core/Chip'
+import { cn } from '../../../utils/cn';
+
+export type GetNfscCardProps = React.ComponentProps<'div'> &
+  Partial<{
+    onClickChip: any,
+    value: number
+  }>;
+
+let cardTitle = "Get $NFSC";
+
+export function GetNfscCard(
+  { onClickChip, value, children, className, ...props }: GetNfscCardProps) {
+    let chips = [20, 50, 100].map(
```

Comment:
this is bad for performance, In react the component function is called multiple times, so in this case on each call a new array `[20, 50, 100]` is constructed and in turn the 3 three chip components are re constructed.
in these kind of cases you have 3 solutions:
- A memoized `ChipGroup` component that takes an array of the numbers.
- useMemo on the array.
- (best for this case) the chips numbers should be a prop and memoize GetNfscCard.
- (easiest fix for this case) define a top-level variable outside the component for the numbers array

Reply:
Now using chipValues as a prop to be memoized by app